src/clamp.js

```diff
@@ -0,0 +1,27 @@
+var _curry3 = require('./internal/_curry3');
+
+/**
+ * Clamps a number within a range.
+ *
+ * @func
+ * @memberOf R
+ * @category Relaction
+ * @sig Ord a => a -> a -> a -> a
```

Comment:
We have (working) tests already.  Because we use `a < b` it will work with Numbers, Strings, Dates, or any type that has a proper `valueOf` method.